### PR TITLE
Support geo location search

### PR DESCRIPTION
**Pull request by toantd90**

### CODE REVIEW

**Comment by toantd90** [comment]
@sideroad Sorry for taking the long time to do this easy feature :(

**Comment by toantd90** [comment]
@sideroad Done!!! :cat:

File: src/creator.js [hunk 1]
```diff
@@ -247,7 +250,13 @@ Creator.prototype = {
       ),
       { minimize: false }
     );
-    schema.index({ id: 1 });
+    let indexAttrs = {};
```
Comment (sideroad) [inline src/creator.js]: is there any reason to remove id index?

**Comment by sideroad** [comment]
thanks!

**Pull request merged**